client/src/features/add-bet/forms/add-bet-form/AddBetForm.tsx

```
@@ -157,16 +160,16 @@ export const AddBetForm = ({
           disabled={disabled}
         />
         <div className="add-bet-buttons">
-          <Button
+          <Button2
             children="Continue"
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)